src/audio-worklet/basic/hello-audio-worklet/main.js

```diff
@@ -3,24 +3,50 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let isModuleLoaded = false;
```

Comment:
You will probably need two different booleans:

`isModuleLoaded`: to guard `addModule()` call.
`isGraphReady`: to guard the node construction/connection logic.

Then the actual `click` event will only care about the context's `resume()` and `suspend()` call.

I believe this is the most efficient way.